src/webui/prefjson.cpp

```diff
@@ -120,7 +120,7 @@ QByteArray prefjson::getPreferences()
     // Global Rate Limits
     data["dl_limit"] = session->globalDownloadSpeedLimit();
     data["up_limit"] = session->globalUploadSpeedLimit();
-    data["enable_utp"] = session->isUTPEnabled();
+    data["enable_utp"] = static_cast<int>(session->btProtocol());
```

Comment:
IMO, we need change field name accordingly.

Reply:
done.